.github/workflows/maven.yml

```
@@ -6,6 +6,8 @@ on:
       - 'main'
       - 'release-v**'
       - 'full-sonar-analysis-**'
+    tags:
+      - 'v*.*.*'
```

We should maybe use a more restrictive rule, such as:

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

---

I checked whether that syntax is supported before, and it seems so! https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

---

I copied it from powsybl-distribution, maybe we could backport that there